.github/workflows/download-data-ingest.yml

```diff
@@ -0,0 +1,32 @@
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  pull_request:
+    branches: [ dev ]
+    paths: [ download-data.sh ]
+    types: [ closed ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  push_download-data.sh:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-18.04
+    steps:
+    # Checks out your repository on the ubuntu-latest machine
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Pushes download-data.sh file
+        uses: dmnemec/copy_file_to_another_repo_action@main
+        env:
+         API_TOKEN_GITHUB: ${{ secrets.GH_TOKEN }}
+        with:
+         source_file: 'download-data.sh'
+         destination_repo: 'PediatricOpenTargets/data-ingestion'
```

Comment:
Thanks for the review @dmiller15! 
It seems the repo has been deleted since I added this PR. @jharenza since we are adding the code to generate OT portal files in this repo itself we do not need this GHA to copy over updated download-data.sh to another repo right?